todo/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+
+
+class Tag(models.Model):
+    name = models.CharField(max_length=100)
+
+    class Meta:
+        ordering = ["name"]
+
+    def __str__(self):
+        return f"{self.name}"
+
+
+class Task(models.Model):
+    content = models.CharField(blank=True, max_length=15)
+    created_at = models.DateTimeField(auto_now_add=True)
+    deadline = models.DateTimeField()
```

Comment:
As required in this project, make this `deadline` field nullable:

```suggestion
    deadline = models.DateTimeField(null=True, blank=True)
```

Don't forget to run `python manage.py makemigrations` and `python manage.py migrate` to apply the changes. If it doesn't work, delete your migration file and run these two commands again.